api/app/types/transcript.ts

```diff
@@ -13,9 +13,16 @@ export interface TranscriptAttributes {
   content: any;
   originalContent: any;
   transcriptHash: string;
+  transcriptUrl: string | null;
   status: string;
   claimedBy?: number | null;
   archivedBy?: number | null;
   archivedAt?: Date | null;
   contentTotalWords: number;
 }
+
+export interface BaseParsedMdContent extends TranscriptAttributes {
+  body: string;
+  [key: string]: string | string[] | any;
```

Comment:
Let's add all the actual fields that the parser returns here, we could avoid some issues if we had them from the beginning.